### DISCUSSION TITLE
リリースノートと関連しそうなCHANGELOG的なものがあっても良いかも

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#152] [12: 新機能の提案]
関連issue: #114 

### [editor.js/docs/CHANGELOG.md](https://github.com/codex-team/editor.js/blob/b355f1673c11ec3c0603303aefaa6a546cf3a377/docs/CHANGELOG.md)

![スクリーンショット 2024-03-30 11 08 45](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/cd96600c-f76f-4c19-b5bf-ce5e7e59a8fe)

**Comment by pss-aileen** [comment]
https://github.com/w3c/reffy/blob/main/CHANGELOG.md

![スクリーンショット 2024-03-30 11 17 08](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/23a65fe8-c615-4f1b-8295-84a5d27b8ffc)

**Comment by kazzyfrog** [comment]
こちら導入しましょう！
#163 にてイシュー化しました！
